metrics-proxy/src/main/resources/templates/cloudwatch_plugin.vm

```diff
@@ -0,0 +1,30 @@
+# Configuration for telegraf agent
+[agent]
+  interval = "${intervalSeconds}s"
+  round_interval = true
+  metric_batch_size = 1000
+  metric_buffer_limit = 10000
+  collection_jitter = "0s"
+  flush_interval = "${intervalSeconds}s"
+  flush_jitter = "0s"
+  precision = ""
+
+# Configuration for AWS CloudWatch output.
+[[outputs.cloudwatch]]
+  region = "$cloudwatchRegion"
+  namespace = "$cloudwatchNamespace"
+#if( $isHosted )
+  access_key = "$cloudwatchSecretKey"
+  secret_key = "$cloudwatchAccessKey"
+#else
+  profile = "$cloudwatchProfile"
```

Comment:
This should only be output if the profile is non-empty. Is that implicitly being taken care of?